kolibri/core/errorreports/tasks.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from django.db import connection
+from django.http import JsonResponse
+
+from .models import ErrorReports
+from kolibri.core.discovery.utils.network.client import NetworkClient
+from kolibri.core.discovery.utils.network.errors import NetworkLocationConnectionFailure
+from kolibri.core.discovery.utils.network.errors import NetworkLocationResponseFailure
+from kolibri.core.discovery.utils.network.errors import NetworkLocationResponseTimeout
+from kolibri.core.tasks.decorators import register_task
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_SERVER_URL = "https://telemetry.learningequality.org"
+
+DEFAULT_PING_JOB_ID = "10"  # Unsure about this value
```

Comment:
This can be left blank. Its optional I think. @rtibbles?

Reply:
The only reason for having this be a set value for the ping back task is to make sure that we never enqueue more than one - this may be needed here, but we can not set it for now and add a fixed id later if we decide it is needed.

Reply:
Okay. So it would be best left unset for now until its use case is clear?